projects/storefrontapp-e2e-cypress/cypress/integration/regression/my-account/payment-methods.e2e-spec.ts

```diff
@@ -17,9 +17,13 @@ describe('Payment Methods', () => {
     before(() => {
       cy.requireLoggedIn();
       cy.reload();
-      cy.visit('/');
 
       cy.server();
+
+      cy.route('GET', '/rest/v2/electronics-spa/cms/pages?*').as('frontPage');
```

Comment:
you can use 

const homePage = waitForPage('homepage', 'getHomePage'); 

above fails once and a while due to waiting for anything pages, but we want the specific page